tools/nni_trial_tool/trial_keeper.py

```diff
@@ -103,6 +104,32 @@ def main_loop(args):
 def trial_keeper_help_info(*args):
     print('please run --help to see guidance')
 
+def check_version(args):
+    trial_keeper_version = None
+    try:
+        trial_keeper_version = pkg_resources.get_distribution('nni').version
+    except:
+        #package nni does not exist
```

Comment:
just catch an expected exception?   and log something here...

Reply:
Yes, changed to catch exception and log warning information.